e2e/cypress/support/api_commands.js

```diff
@@ -89,6 +89,20 @@ Cypress.Commands.add('apiCreateWebhook', (hook = {}, isIncoming = true) => {
     });
 });
 
+Cypress.Commands.add('apiGetWebhook', (hookId, isIncoming = true) => {
+    const hookUrl = isIncoming ? `/api/v4/hooks/incoming/${hookId}` : `/api/v4/hooks/outgoing/${hookId}`;
+    const options = {
+        url: hookUrl,
+        headers: {'X-Requested-With': 'XMLHttpRequest'},
+        method: 'GET',
+        failOnStatusCode: false,
+    };
+
+    return cy.request(options).then((response) => {
+        return cy.wrap(response);
+    });
+});
```

Comment:
I'm fine as is but we should no longer adding any API commands in this file. All new API commands should move to `e2e/cypress/support/api/` folder and should be added into a file that corresponds to the structure of https://api.mattermost.com/#tag/webhooks. So in this case, this should be at new `e2e/cypress/support/api/webhooks.js`. It should have type as well to be located at new `e2e/cypress/support/api/webhooks.d.ts`. Another is it should be of two separate API commands that map directly to their specific API endpoints:
- `apiGetIncomingWebhook ` for https://api.mattermost.com/#tag/webhooks/paths/~1hooks~1incoming~1{hook_id}/get
- `apiGetOutgoingWebhook ` for https://api.mattermost.com/#tag/webhooks/paths/~1hooks~1outgoing~1{hook_id}/get

Reply:
Sorry it was my suggestion he followed just to make it work initially. But yes, please follow the current convention.